tutorials/19_get_follower_and_following_list/public/index.html

```diff
@@ -0,0 +1,76 @@
+<html>
+    <head>
+        <title>Get Follower and Following List</title>
+        <script src="bundle.js"></script>
+    </head>
+    <body>
+    	<div class="container" id="content"><br/>
+            <h4>Get list of Followers and Following for User</h4>
+            <form>
+                <div class="form-group row">
+                    <label for="username" class="col-sm-2 col-form-label">
+                        Username
+                    </label>
+                    <div class="col-sm-10">
+                        <input
+                            id="username"
+                            type="text"
+                            class="form-control-plaintext"
```

Comment:
Counter-productive stylishness. 

Remove `-plaintext`. It's good that the field is editable. It should be easily understood that it's editable.